integration/common/openlineage/common/provider/dbt/processor.py

```diff
@@ -544,6 +545,8 @@ def extract_namespace(self, profile: Dict) -> str:
             return f"databricks://{profile['host']}"
         elif self.adapter_type == Adapter.SQLSERVER:
             return f"mssql://{profile['server']}:{profile['port']}"
+        elif self.adapter_type == Adapter.DREMIO:
+            return f"dremio://{profile['host']}:{profile['port']}"
```

Comment:
I know basically nothing about dremio but [this](https://docs.dremio.com/current/sonar/client-applications/clients/dbt/#configurations) points to `software_host` not `host`.

Reply:
@JDarDagran , thanks for highlighting it, I'll handle this situation in next PR. Also, not sure why this current PR even after getting merged didn't passed all the test cases (flink specifically). Even though this is not related to flink